vslib/src/tests.cpp

```diff
@@ -297,7 +297,7 @@ void test_set_stats_via_redis()
     sai_port_stat_t ids[1] = { SAI_PORT_STAT_IF_IN_ERRORS };
     uint64_t counters[1] = { 0 } ;
 
-    SUCCESS(sai_metadata_sai_port_api->get_port_stats(ports.at(0), 1, ids, counters));
+    SUCCESS(sai_metadata_sai_port_api->get_port_stats(ports.at(0), 1, (const sai_stat_id_t *)ids, counters));
```

Comment:
previously we had each type for each stats, and cast was not necessary

Reply:
Yes, but I'm adapting to the new SAI APIs.
I don't know what was the reason behind changing all of them to the same type, but now this is a necessary change.